documentation/feature/goggles.md

```diff
@@ -7,17 +7,15 @@ parent: wiki
 
 ## Overview
 
-### Visual Effects for eyewear
-Adds color tint to sunglasses and other eyewear. Causes raindrops to appear on 
+### Visual Effects for eye wear
```

Comment:
When written apart it sounds more like "eye fatigue". It was correct before; see:
* [Wikipedia](http://en.wikipedia.org/wiki/Eyewear)
* [Dictionary](http://www.thefreedictionary.com/eyewear)